in-cluster/default.yaml

```diff
@@ -371,6 +371,19 @@ spec:
           - pass:
               message: Goldpinger can communicate properly
         regexGroups: '"OK": ?(?P<OK>\w+)'
+    - textAnalyze:
+        checkName: Etcdserver Database Size Exceeded
+        exclude: ""
+        ignoreIfNoFiles: true
+        fileName: *
```

Comment:
lgtm, just curious if this works? never knew if so.

Reply:
yeah I have never tried it before either, but it works across all the files in the bundle